roles/internal/grok/tasks/install.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Clone Grok
+  git:
+    repo: https://github.com/GrokImageCompression/grok.git
+    dest: "{{ grok_clone_directory }}"
+    version: "{{ grok_version_tag }}"
+
+- name: Make build directory
+  file:
+    path: "{{ grok_clone_directory }}/build"
+    state: directory
+    mode: 0755
+
+- name: Cmake grok
+  cmake:
+    binary_dir: "{{ grok_clone_directory }}/build"
+    source_dir: "{{ grok_clone_directory }}"
+    build_type: Release
+      
+- name: Make grok
+  make:
+    chdir: "{{ grok_clone_directory }}/build"
+
+- name: Install grok
```

Comment:
Did you run the role twice to make sure this is idempotent? Taking a quick glance at the cmake library that is included, I don't think that this will be idempotent without the creates statement, but I could be reading the code wrong and I haven't actually run it yet.

You can tell if its idempotent by running it twice and seeing if ansible reports any tasks as changed.

Reply:
So it does seem to have 'changed' it. But I also realized reading the code that he has a _creates_ option, so I'll try adding that.